pyscript.py

```diff
@@ -0,0 +1,26 @@
+import re
+
+def alphanumeric_compress(text):
+    # Remove all numbers
+    pattern = r'[0-9]'
+    string = re.sub(pattern, '', text)
+    # Init vars
+    res = ""
+    count = 1
+    # Add first character to res
+    res += string[0]
+    # Count alphanumeric number for every character
+    for i in range(len(string)-1):
+        if(string[i] == string[i+1]):
+            count += 1
+        else:
+            if(count > 1):
+                res += str(count)
+            res += string[i+1]
+            count = 1
```

Comment:
Try to avoid the 3rd level of control structures. They get delicate fast and are generally more difficult to maintain. The most important thing is that the code run and is clear enough to read (which this is).